### PR TITLE
[112] - Добавить скрипты для Ci iOS 

### PR DESCRIPTION
**Pull request by KirBorisov96**

### CODE REVIEW

File: example/ios/Runner/Info.plist [hunk 1]
```diff
@@ -21,7 +23,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(FLUTTER_BUILD_NUMBER)</string>
+	<string>1.0</string>
```
Comment (Andyatkov) [inline example/ios/Runner/Info.plist:26]: Мне кажется что FLUTTER_BUILD_NUMBER?

Comment (KirBorisov96) [reply example/ios/Runner/Info.plist:26]: ага, надо поправить

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 15
Version: 1670939605
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/15

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 16
Version: 1670941321
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/16

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 17
Version: 1670941782
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/17

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 18
Version: 1670942037
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/18

**Review by Andyatkov — APPROVED** [review]

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 19
Version: 1670949415
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/19

**Review by Almazovivan — APPROVED** [review]

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 20
Version: 1670964668
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/112-ci-ios
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/20

**Pull request merged**